### DISCUSSION TITLE
Deployments perpetually queued

### DISCUSSION THREAD

**Discussion by ghost** [vercel/vercel#5208] [Help] [answered]
We're getting deployments stuck permanently in a queued state. I can verify that there are no other deployments causing us to exceed the plan's concurrency limit.

**Comment by paulogdm** [comment, marked as answer]
Please get in touch with support so we can address your concern: https://vercel.com/support/request

**Comment by alesanchezr** [comment]
This is happening to me as well, I decided to cancel a deployment in the middle and now its been 2 hours stuck at "Queued", there is nothing I can do, I event paid for more concurrent deployments and nothing happened.

**Comment by homerjam** [comment]
Update: I had contacted support already but heard back yesterday - the problem relates to an issue with the Nuxt builder and the new monorepo support. Vercel are aware and working on it.
